tests/sles4sap/hana_cluster.pm

```diff
@@ -90,15 +117,15 @@ sub run {
             # sr_state is not online after 90 seconds. Start system again and retry
             assert_script_run $start_cmd;
             sleep bmwqemu::scale_timeout(10);
-            assert_script_run "until su - $sapadm -c 'hdbnsutil -sr_state' | grep -q 'online: true' ; do sleep 1 ; done";
+            assert_script_run "until su - ${sapadm} -c 'hdbnsutil -sr_state' | grep -q 'online: true' ; do sleep 1 ; done";
         }
 
         # Synchronize the nodes
-        barrier_wait "HANA_CREATED_CONF_$cluster_name";
+        barrier_wait "HANA_CREATED_CONF_${cluster_name}";
     }
 
     # Synchronize the nodes
-    barrier_wait "HANA_LOADED_CONF_$cluster_name";
+    barrier_wait "HANA_LOADED_CONF_${cluster_name}";
```

Comment:
Added `{}` again. Rule of thumb in my case (could be personal preference):

Always reference the variables without `{}`, unless the variable name leads to problems when formatting a string. For example, if you have `$var = 'test'` and want to do `print "$var_number_$number"`, then you need to do `${var}_number_$number`, otherwise Perl will interpret `$var_number_` as the actual name and fail:

```
root@mango:/var/lib/openqa/tests/sle [master|…12] > perl
use strict;
use warnings;
my $var = 'test';
foreach my $number (1..3) { print "$var_number_$number\n" }
Global symbol "$var_number_" requires explicit package name (did you forget to declare "my $var_number_"?) at - line 4.
```
vs.
```
root@mango:/var/lib/openqa/tests/sle [master|…12] > perl
use strict;
use warnings;
my $var = 'test';
foreach my $number (1..3) { print "${var}_number_$number\n" }
test_number_1
test_number_2
test_number_3
```
Elsewhere in os-autoinst-distri-opensuse, referencing the variables with `{}` is rarely used.

Reply:
Curly braced variable is safely enclosed variable, you known, from my personal perspective it look better when variable handling is done same way. But OFC if it's a problem, I could live with it.